src/de/jost_net/JVerein/gui/menu/FreieFormulareMenu.java

```diff
@@ -0,0 +1,48 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.gui.menu;
+
+import java.rmi.RemoteException;
+
+import de.jost_net.JVerein.Einstellungen;
+import de.jost_net.JVerein.gui.action.FreiesFormularAction;
+import de.jost_net.JVerein.keys.FormularArt;
+import de.jost_net.JVerein.rmi.Formular;
+import de.willuhn.datasource.rmi.DBIterator;
+import de.willuhn.jameica.gui.parts.CheckedContextMenuItem;
+import de.willuhn.jameica.gui.parts.ContextMenu;
+
+/**
+ * Kontext-Menu zu den Mails.
+ */
+public class FreieFormulareMenu extends ContextMenu
+{
+
+  public FreieFormulareMenu() throws RemoteException
```

Comment:
Du könntest hier auch den Iterateor übergeben, dann ist weniger Code nötig und nicht doppelt die DB Abfrage:
`public FreieFormulareMenu(DBIterator<Formular> it) throws RemoteException`

Reply:
Gute Idee, mache ich